diff --git a/dist/tool.d.ts b/dist/tool.d.ts
index 865fde036398b9afa5141a8533f89ae5bd49947e..02cf7f76b120c038681f60077f791157ffe904a7 100644
--- a/dist/tool.d.ts
+++ b/dist/tool.d.ts
@@ -33,6 +33,10 @@ export type FunctionTool<Context = UnknownContext, TParameters extends ToolInput
      * A JSON schema describing the parameters of the tool.
      */
     parameters: JsonObjectSchema<any>;
+    /**
+     * The original parameters of the tool.
+     */
+    originalParameters: ZodObject<any>;
     /**
      * Whether the tool is strict. If true, the model must try to strictly follow the schema (might result in slower response times).
      */
diff --git a/dist/tool.mjs b/dist/tool.mjs
index d60b5d6f43a73a1e53ccee441873d8496d9a0255..40cc6c4d5ec088999fe987f8939bc69bf1865633 100644
--- a/dist/tool.mjs
+++ b/dist/tool.mjs
@@ -140,6 +140,7 @@ export function tool(options) {
         name,
         description: options.description,
         parameters,
+        originalParameters: options.parameters,
         strict: strictMode,
         invoke,
         needsApproval,
